client/packages/common/src/ui/components/navigation/Tabs/TabKeybindings.tsx

```diff
@@ -4,6 +4,7 @@ interface TabKeybindingsProps<T> {
   tabs: T[];
   onAdd?: () => void;
   setCurrentTab: React.Dispatch<React.SetStateAction<T>>;
+  dependencies?: any[];
```

Comment:
unknown[] might be the more fitting type?

Reply:
perfect, thanks!